libs/core/src/lib/resizable-card-layout/resizable-card-layout.component.scss

```diff
@@ -0,0 +1,2255 @@
+/*!
+ * Fundamental Library Styles v0.17.0-rc.30
+ * Copyright (c) 2021 SAP SE or an SAP affiliate company.
+ * Licensed under Apache License 2.0 (https://github.com/SAP/fundamental-styles/blob/main/LICENSE)
+ */
+ @charset "UTF-8";
+ [class*=sap-icon] {
```

Comment:
Why do you need to include the icon CSS here? You should use `@import` instead.

Reply:
Resizable-card-layout css is not yet released.
as per core team suggestion, i pasted dist/resizable-card-layout.css here.
later when it is released; this can be replaced.

Reply:
But why are you copying the `sap-icon` CSS? Doesn't that already exist in the `icon.css` - it should be imported in the `resizable-card-layout.scss` too.

Reply:
@DeepakSap14 ngx is on top of the latest styles, please, clean up this